api/Element.json

```diff
@@ -1193,47 +1210,103 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "getElementsByTagName(*)": {
```

Comment:
Sorry about the unhelpful schema error messages. I think this is about identifiers which can't have characters like "(" or "*".

This should be something like:
 
```
"all_elements_selector": {
  "__compat": {
    "description": "getElementsByTagName(*)",
    "support": {}
  }
}
```